docs/readmes/nms/federation.md

```diff
@@ -0,0 +1,55 @@
+---
+id: federation
+title: Federation
+hide_title: true
+---
+# Federated Networks
+
+Magma implements a low-friction, single point of integration between an MNO's mobile core and access gateways through federation networks. NMS enables management of the federated networks and their gateways.
+
+Magma has two important concepts on federation:
+
+### **Federation Network/** Federation Gateway (FGW)
+
+Federation network comprises of the federation gateways. The Federation Gateway integrates the Operators core network with using standard 3GPP interfaces to existing MNO components. It acts as a proxy between the Magma AGW and the operator’s network and facilitates core functions, such as authentication, data plans, policy enforcement, and charging.
+
+### **Federated LTE Network**
+
+Federated LTE network is the network containing AGWs which are managed through the federated networks. When configuring an integration with LTE nodes, it is necessary to link these two entities as described in the following sections.
```

Comment:
Not sure how we call to the `AGW` on the rest of the system, but maybe it would be good to say `LTE Access Gateway` so they can match with the name they see on nms